File: Server/src/main/java/com/api/server/foodb/FoodcomexUserDatum.java
package com.api.server.foodb;

import jakarta.persistence.*;
import jakarta.validation.constraints.NotNull;
import jakarta.validation.constraints.Size;
import lombok.Getter;
import lombok.Setter;

import java.time.Instant;

@Getter
@Setter
@Entity
@Table(name = "foodcomex_user_data")
public class FoodcomexUserDatum {
    @Id
    @Column(name = "id", nullable = false)
    private Integer id;

    @Size(max = 255)
    @NotNull
    @Column(name = "role", nullable = false)
    private String role;

    @Size(max = 255)
    @Column(name = "gender")
    private String gender;

    @Size(max = 255)
    @Column(name = "position")
    private String position;

    @Size(max = 255)
    @Column(name = "institution")
    private String institution;

    @Size(max = 255)
    @Column(name = "department")
    private String department;

    @Size(max = 255)
    @Column(name = "address")
    private String address;

    @Size(max = 255)
    @Column(name = "city")
    private String city;

    @Size(max = 255)
    @Column(name = "country")
    private String country;

    @Size(max = 255)
    @Column(name = "phone_number")
    private String phoneNumber;

    @Size(max = 255)
    @Column(name = "zip_code")
    private String zipCode;

    @Size(max = 255)
    @Column(name = "institution_type")
    private String institutionType;

    @Size(max = 255)
    @Column(name = "area_of_research")
    private String areaOfResearch;

    @Lob
    @Column(name = "area_of_research_detailed")
    private String areaOfResearchDetailed;

    @Lob
    @Column(name = "activities")
    private String activities;

    @Lob
    @Column(name = "interests")
    private String interests;

    @Lob
    @Column(name = "purpose_of_use")
    private String purposeOfUse;

    @Column(name = "created_at")
    private Instant createdAt;

    @Column(name = "updated_at")
    private Instant updatedAt;

    @Column(name = "admin_user_id")
    private Integer adminUserId;

}

File: DotNetServer/DotNetServer/Entities/Foodb/FoodcomexUserDatum.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace DotNetServer.Entities.Foodb;

[Table("foodcomex_user_data")]
public partial class FoodcomexUserDatum
{
    [Key] [Column("id")] public int Id { get; set; }

    [Column("role")] [StringLength(255)] public string Role { get; set; } = null!;

    [Column("gender")] [StringLength(255)] public string? Gender { get; set; }

    [Column("position")]
    [StringLength(255)]
    public string? Position { get; set; }

    [Column("institution")]
    [StringLength(255)]
    public string? Institution { get; set; }

    [Column("department")]
    [StringLength(255)]
    public string? Department { get; set; }

    [Column("address")]
    [StringLength(255)]
    public string? Address { get; set; }

    [Column("city")] [StringLength(255)] public string? City { get; set; }

    [Column("country")]
    [StringLength(255)]
    public string? Country { get; set; }

    [Column("phone_number")]
    [StringLength(255)]
    public string? PhoneNumber { get; set; }

    [Column("zip_code")]
    [StringLength(255)]
    public string? ZipCode { get; set; }

    [Column("institution_type")]
    [StringLength(255)]
    public string? InstitutionType { get; set; }

    [Column("area_of_research")]
    [StringLength(255)]
    public string? AreaOfResearch { get; set; }

    [Column("area_of_research_detailed", TypeName = "text")]
    public string? AreaOfResearchDetailed { get; set; }

    [Column("activities", TypeName = "text")]
    public string? Activities { get; set; }

    [Column("interests", TypeName = "text")]
    public string? Interests { get; set; }

    [Column("purpose_of_use", TypeName = "text")]
    public string? PurposeOfUse { get; set; }

    [Column("created_at", TypeName = "datetime")]
    public DateTime? CreatedAt { get; set; }

    [Column("updated_at", TypeName = "datetime")]
    public DateTime? UpdatedAt { get; set; }

    [Column("admin_user_id")] public int? AdminUserId { get; set; }
}